src/modules/flow/http-client/http-client.json

```diff
@@ -777,6 +777,161 @@
       ]
     },
     {
+      "name": "http-client/create-url",
+      "private_data_type": "create_url_data",
+      "category": "http-generic",
+      "description": "Creates an encoded URL",
+      "url": "http://solettaproject.org/doc/latest/node_types/http-client/create-url.html",
+      "methods": {
+        "close": "create_url_close",
+        "open": "create_url_open"
+      },
+      "options": {
+        "members": [
+          {
+            "data_type": "string",
+            "default": "http",
+            "description": "The URL protocol",
+            "name": "scheme"
+          },
+          {
+            "data_type": "int",
+            "default": 80,
```

Comment:
make it default to zero, so if we change the protocol/scheme we won't be getting 80 as their port ;-)